configure

```diff
@@ -452,12 +460,16 @@ if test "$gcc" -eq 1 && ($cc $CFLAGS -c $test.c) >> configure.log 2>&1; then
         LDSHARED=${LDSHARED-"$cc"}
         LDSHAREDFLAGS="-shared -Wl,--out-implib,${IMPORTLIB}"
         LDSHAREDLIBC=""
-        DEFFILE='win32/${LIBNAME2}.def'
+        if test $gzfileops -eq 0; then
```

Comment:
I'm not sure what some of these visibility, RCFLAGS, gzfileops changes are. Perhaps they should be separate commits.

Reply:
MinGW doesn't support visibility attribute, but gives only warning, not error, so it makes sense to not check it for configure as we don't parse the compiler output like cmake does.

´RCFLAGS` adds binary/output directory to the include path for `windres` as it doesn't do it automatically. Otherwise it won't find `zlib.h` or `zlib-ng.h`.